Sources/BraveWallet/Crypto/Stores/PortfolioStore.swift

```diff
@@ -186,10 +186,18 @@ struct BalanceTimePrice: DataPoint, Equatable {
   }
 }
 
+struct BalanceDifference: Equatable {
+  let priceDifference: String
+  let percentageChange: String
+  let isBalanceUp: Bool
+}
+
 /// A store containing data around the users assets
 public class PortfolioStore: ObservableObject, WalletObserverStore {
   /// The dollar amount of your portfolio
   @Published private(set) var balance: String = "$0.00"
+  ///
```

Comment:
forgot to add the comment?

Reply:
Added comment in https://github.com/brave/brave-ios/pull/8140/commits/8ff2febc76b935a459487f20c4421f7a334d6125